concert/helpers.py

```diff
@@ -232,3 +234,59 @@ def is_iterable(item):
         return True
     except Exception:
         return False
+
+
+def arange(start, stop, step):
```

Comment:
Maybe you can name it so that it's clear the stripping occurs...

Reply:
But it strips the unit only internally. From the outside it looks like the numpy function but with proper unit handling. I think if one combination of numpy and pint this was possible a while ago.